azure-pipelines.yml

```diff
@@ -0,0 +1,38 @@
+# File: hauki-admin-ui/azure-pipelines.yml
+
+# Continuous integration (CI) triggers cause a pipeline to run whenever you push 
+# an update to the specified branches or you push specified tags.
+trigger:
+  branches:
+    include:
+    - master
+    - feature/*
+  paths:
+    exclude:
+    - README.md
+
+# Pull request (PR) triggers cause a pipeline to run whenever a pull request is 
+# opened with one of the specified target branches, or when updates are made to 
+# such a pull request.
+#
+# GitHub creates a new ref when a pull request is created. The ref points to a 
+# merge commit, which is the merged code between the source and target branches 
+# of the pull request.
+pr:
```

Comment:
After discussing with Jussi Arpalahti we decided the pull-request pipeline would be a separate from this. So according to current plan we would not need this pipeline to be triggered from pull-requests and therefore this could be removed?